qemu/tests/single_driver_install.py

```diff
@@ -144,7 +145,7 @@ def run(test, params, env):
             vm.destroy()
             vm.create()
             vm = env.get_vm(params["main_vm"])
-            session = vm.wait_for_login()
+            session = vm.wait_for_serial_login()
```

Comment:
@heywji This is a workaround, please take some note here.

Reply:
OK!